isic_challenge_scoring/types.py

```diff
@@ -9,13 +9,13 @@ class ScoreException(Exception):
 SeriesDict = Dict[str, float]
 DataFrameDict = Dict[str, SeriesDict]
 RocDict = Dict[str, List[float]]
-ScoreDict = Dict[str, Union[float, SeriesDict, DataFrameDict, Dict[str, RocDict]]]
+ScoreDict = Dict[str, Union[Optional[float], SeriesDict, DataFrameDict, Dict[str, RocDict]]]
```

Comment:
```suggestion
ScoreDict = Dict[str, Union[float, Optional[float], SeriesDict, DataFrameDict, Dict[str, RocDict]]]
```

Unless the type checker doesn't like this, it's more accurate.